incuna-sass/helpers/_image-replace.sass

```diff
@@ -9,6 +9,3 @@
         display: block
         width: 0
         height: 100%
-    @if support-legacy-browser("ie", "7")
-        .lt-ie8 &
-            text-indent: -9999px
```

Comment:
According to what I could read in the article from perry (posted yesterday) `text-indent` is accessible (apart from RTL languages)

http://snook.ca/archives/html_and_css/hiding-content-for-accessibility